books/free-programming-books-fr.md

```diff
@@ -203,24 +203,24 @@
 
 #### PGF/TikZ
 
-* [TikZ pour l'impatient](http://math.et.info.free.fr/TikZ/) - Gérard Tisseau et Jacques Duma
+* [TikZ pour l'impatient](http://math.et.info.free.fr/TikZ/) - Gérard Tisseau, Jacques Duma
 
 
 #### TeX
 
 * [Apprendre à programmer en TeX](https://ctan.org/pkg/apprendre-a-programmer-en-tex) - Christian Tellechea
-* [TeX pour l'Impatient](http://www.apprendre-en-ligne.net/LaTeX/teximpatient.pdf) - Paul Abrahams, Kathryn Hargreaves, and Karl Berry, trad. Marc Chaudemanche (PDF)
```

Comment:
the translator notation should stay

Reply:
I understand, but in the most of files 98% the traducer is part of the names without any role label.

There are in `hu` `it` `en` `fr`

is `trad.` enough for all file locales?

Reply:
one possibility is to use `(trl)` https://www.loc.gov/marc/relators/relaterm.html

Reply:
Recovered at 0a4dbc179e4b2f6346fc15579d7c5edec1b3433e

Reply:
> one possibility is to use `(trl)` or `trl.` [loc.gov/marc/relators/relaterm.html](https://www.loc.gov/marc/relators/relaterm.html)

Applied in 1833c6db7a3d67788f5bda88506d7071c16cd61c